posthog/temporal/data_modeling/run_workflow.py

```diff
@@ -262,9 +262,11 @@ async def handle_model_ready(model: ModelNode, team_id: int, queue: asyncio.Queu
         if model.selected is True:
             team = await database_sync_to_async(Team.objects.get)(id=team_id)
             await materialize_model(model.label, team)
-    except Exception:
+    except Exception as err:
+        await logger.aexception("Failed to materialize model %s due to error: ", model.label, str(err))
```

Comment:
Exception handler will log a traceback, so we are not losing anything by adding the exception name with `str(err)`.